nixos/modules/config/zram.nix

```diff
@@ -92,6 +99,46 @@ in
           as there's no gain from keeping them in RAM.
         '';
       };
+
+      use-optimal-sysctls = lib.mkOption {
```

Comment:
Generally I think we use javaCase for options? Also, "optimal" is a big word. "Recommended", maybe? Also, maybe something more generic than "sysctls", just for future potential extensibility?

Reply:
That sounds OK, unless anyone has better ideas.